src/Application.php

```diff
@@ -19,17 +19,17 @@ class Application
     /**
      * @var ServerRequestFactory
      */
-    private $requestFactory;
+    private $_requestFactory;
```

Comment:
No, we are not using `_` anymore.

Reply:
Ok no problem, I already make the changes in the whole code.